scripts/annotation.js

```diff
@@ -14,38 +14,53 @@ $.urlParam = function (a) {
 
 //Our stuff
 $(document).ready( ()=> {
-    $("#name").text($.urlParam('name'));
-    $("#structure").text($.urlParam('structure'));
-    index = $.urlParam('index');
-    $("#image").attr('src', '/images?index=' + index);
+    const name = $.urlParam('name');
+    const structure = $.urlParam('structure');
+    let index = $.urlParam('index');
+    let image_div = $("#image");
+    $("#name").text(name);
+    $("#structure").text(structure);
+    image_div.attr('src', '/images?index=' + index);
   
     document.onkeyup = function (event) {
-        var e = (!event) ? window.event : event;
+        let e = (!event) ? window.event : event;
+        let choice;
         switch (e.keyCode) {
             //left arrowkey
             case 37:
-                prevImage();
+                // prevImage();
+                choice = false;
                 break;
             //right arrowkey
             case 39:
-                nextImage();
+                // nextImage();
+                choice = true;
                 break;
         }
+        $.post("annotate", {imageid : index, user: name, annotation: choice, feature: structure})
+            .done( data => {
+                index = nextImage(index)
+            })
+            .fail( err => {
+                alert("Something went wrong...\n" + err.responseText);
+            })
     };
 
-    $("#image").click( ()=> {
-        nextImage();
+    image_div.click( ()=> {
```

Comment:
I'm going to make a couple of divs on each side of the polaroid that users can click on and it will have the same behavior as the arrow keys being pressed.